server/src/endpoints/minidump.rs

```diff
@@ -94,6 +113,19 @@ where {
                 None => return Err(BadStoreRequest::MissingMinidump),
             }
 
+            for field_name in &[ITEM_NAME_BREADCRUMBS1, ITEM_NAME_BREADCRUMBS2] {
+                if let Some(item) = envelope.get_item_by_mut(|item| item.name() == Some(field_name))
```

Comment:
_nit_: Could introduce a convenience function like the non-mutable version.

Reply:
I am already unhappy with the combinatorial explosion of convenience functions. Let's just draw the line here even though it's arbitrary

Reply:
Then we should also remove the others. It’s inconsistent otherwise.

Reply:
I removed them now 😅 

I think if we want to have them again we want shortcuts for building predicate functions, not for `get_item_by_mut`, `take_item_by` and `get_item_by`

Reply:
Look at this slick haskell syntax:

```haskell
a = [1,2,3]
filter (2==) a  -- [2]
```